controllers/authorizeService.js

```diff
@@ -0,0 +1,113 @@
+var AccountingService = require('../db/schemas/accountingService'),
+     config = require('../config'),
+     uuid = require('node-uuid');
+
+var authorizeService = (function () {
+
+    /**
+     * Generates am aìKey.
+     */
+    var generateApiKey = function () {
+        var apiKey = uuid.v4();
+
+        return apiKey;
+    };
+
+    /**
+     * Check if the remote cliente is the WStore; otherwise return an error.
+     */
+    var checkRemoteClient = function (ip) {
+        var storeHostname = config.appHost;
+        var remoteHostname = ip.replace(/^.*:/, ''); // Parse IPv4 embedded in IPv6
+
+        return remoteHostname === storeHostname;
+    }
+
+    /**
+     * Generates and send an apiKey for the url service specifed in the request body. The apiKey is saved in "uncommitted" state.
+     *
+     * @param  {Object} req     Incoming request.
+     * @param  {Object} res     Outgoing object.
+     */    
+    var getApiKey = function (req, res) {
+        // Check if request is from WStore
+        var fromWStore = checkRemoteClient(req.ip);
```

Comment:
You can call the function directly in the if statement as you are not using this variable anymore...